# HG changeset patch
# User Max Brister <max@2bass.com>
# Date 1375572224 21600
#      Sat Aug 03 17:23:44 2013 -0600
# Node ID bb60b1bb47cff2ea2604df4de01a5d275a6d2106
# Parent  36fd98b9ce487fb8cbf7f1aaa7fa5fc36651c735
view.m: Fix args undefined error when calling with two arguments

diff --git a/scripts/plot/view.m b/scripts/plot/view.m
--- a/scripts/plot/view.m
+++ b/scripts/plot/view.m
@@ -74,18 +74,18 @@ function [azimuth, elevation] = view (va
       el = 90;
     elseif (x == 3)
       az = -37.5;
       el = 30;
     else
       print_usage ();
     endif
   elseif (length (varargin) == 2)
-    az = args{1};
-    el = args{2};
+    az = varargin{1};
+    el = varargin{2};
   endif
 
   if (nargin > 0)
     set (hax, "view", [az, el]);
   else
     if (nargout == 1)
       azimuth = [az, el];
     elseif (nargout == 2)
